### ISSUE TITLE
Move and clarify the "Safety versus comfort" section

### ISSUE THREAD

**Issue by dbooth-boston** [w3c/PWETF#233]
This issue separates out (and supersedes) suggestion 12 from https://github.com/w3c/PWETF/issues/228 .

Section 3.2.1 (Safety versus comfort) is currently a subsection of "Unacceptable behavior", and reads:

[[

3.2.1. Safety versus Comfort

This Code prioritizes the safety of individuals, particularly those in marginalized communities, over the comfort of others, for example in situations involving:

- "Reverse" -isms, including "reverse racism," "reverse [sexism](https://www.w3.org/Consortium/cepc/cepc-20200716/#dfn-sexism)," and "cisphobia".
- Reasonable communication of boundaries, such as “leave me alone,” “go away,” or “I’m not discussing this with you”.
- Communication in a tone you don’t find congenial.
- Criticisms of racist, sexist, [cissexist](https://www.w3.org/Consortium/cepc/cepc-20200716/#dfn-cissexist), or otherwise oppressive behavior or assumptions.

]]

SUGGESTION: Move this section to become a new section 4.3, and reword it as follows:

[[

4.3 Safety versus Comfort

This Code prioritizes the safety of individuals, particularly those in marginalized communities, over the comfort of others. For example:

- Although a curt response like "leave me alone", “go away,” or “I’m not discussing this with you” would normally be a violation of this code, such a response is justifiable (as self defense) when the respondent's safety feels threatened by an aggressor, and should be immediately respected.

- The Ombud may not act on claims of "reverse"-isms, including "reverse racism," "reverse sexism," and "cisphobia", that arise out of reasonable efforts to correct a larger injustice.

- Efforts to prevent or correct oppressive behavior, such as criticizing racist, sexist or cissexist behavior, are permissible even if they make other participants uncomfortable.

]]

EXPLANATION: 1. This section is more about enforcement, and how Ombuds should respond to complaints, than it is about "unacceptable behavior", so section 4 seems like a more appropriate place for it. 2. The "go away" example needed more contextual explanation to avoid implying that it is normally okay to respond "go away" to someone. 3. Similarly, the other examples needed a little more context.

**Comment by nigelmegitt** [comment]
> when the respondent's safety feels threatened

I'd rephrase this to be active, i.e.: "when the respondent feels their safety is threatened"

> * The Ombud may not act on ...

I'd steer away from "may not" because it is ambiguous: in ordinary English it means what spec writers would have as "MUST NOT", but W3C folk are likely to read it as "MAY". I'm not actually sure which one you mean here, but suspect that this would work better:

[[
* The Ombud may choose not to act on ...
]]

**Comment by dbooth-boston** [comment]
@nigelmegitt good ideas, though as one more friendly amendment, I suggest using "decide" instead of "choose" (to sound more considered and less arbitrary), as in: "The Ombud may decide not to act on ..."

**Comment by wareid** [comment]
Closing as we've merged a PR for this section.

**Comment by dbooth-boston** [comment]
1. When closing an issue, it's helpful to include a link to the PR(s) and/or minutes that explain why the issue was closed (if it isn't explained directly in the issue), so that readers can follow the group's reasoning.  I see https://github.com/w3c/PWETF/pull/251

2. It is premature to close this issue.  Although https://github.com/w3c/PWETF/pull/251 looks good as far as it goes, the main point of this issue has not yet been addressed.  As https://github.com/w3c/PWETF/issues/263 also points out, more clarification is still needed in the Safety vs Comfort section https://github.com/w3c/PWETF/blob/da79c51e018605c9fc7c8beedbb977c971087e20/index.html#L366 .  Since the "reverse-isms" bullet is addressed in 263, I'll only address the other bullets here.

As previously explained in https://github.com/w3c/PWETF/issues/228 , the examples lack context.  Context is needed to give the reader insight about how the safety-versus-comfort principle applies to the given examples.  @Tzviya gave some excellent explanation of context in https://github.com/w3c/PWETF/issues/228#issuecomment-1372316685 .  I tried to incorporate that context in suggestion 12 at https://github.com/w3c/PWETF/issues/228#issuecomment-1407551041 , but AFAICT the suggestion seems to have gone unnoticed:

> Although a curt response like "leave me alone", “go away,” or “I’m not discussing this with you” would normally be a violation of this code, such a response is justifiable (as self defense) when the respondent's safety feels threatened by an aggressor, and should be immediately respected.

I would now amend that suggestion to shorten it and also cover the bullet about communicating in a non-congenial tone:

> Although a non-congenial tone or curt response like "leave me alone", “go away,” or “I’m not discussing this with you” would normally be a violation of this code, such a response is justifiable (as self defense) when the respondent's safety feels threatened.

Others might come up with better wordings.  The point is that, without context, it isn't clear what these bullets are intending to convey.
